ocrd_tesserocr/segment_region.py

```diff
@@ -12,90 +24,242 @@
     MathsRegionType,
     SeparatorRegionType,
     NoiseRegionType,
+    to_xml)
+from ocrd_models.ocrd_page_generateds import TableRegionType
+from ocrd import Processor
 
-    to_xml
+from .config import TESSDATA_PREFIX, OCRD_TOOL
+from .common import (
+    image_from_page,
+    save_image_file,
+    membername
 )
-from ocrd import Processor
 
-from ocrd_tesserocr.config import TESSDATA_PREFIX, OCRD_TOOL
+TOOL = 'ocrd-tesserocr-segment-region'
+LOG = getLogger('processor.TesserocrSegmentRegion')
+FILEGRP_IMG = 'OCR-D-IMG-CROP'
 
-log = getLogger('processor.TesserocrSegmentRegion')
+PADDING = 8 # extend detected region rectangles by how many (true) pixels?
```

Comment:
As a parameter? (It was 4 Pixels above...)

Reply:
Sure!